### PR TITLE
fix : (configparser) stop baseUrl override & 1 new playground for monorepo

### PR DESCRIPTION
**Pull request by vegarringdal**
### Do not merge yet....

**FIX:**
Fix path override in config parser. 
No errors or anything atm, since I have one thing bugging in samples Ive added.

**Playground**
Created and mini sample under playground.
Added fuse-box-typechecker to be able to see that all works.
Both sampels have own tsconfig and base file to test better.

Project 1:
`ts-node -T ./playground/monorepo_alt/samples/project1/fuse`
This have baseurl in base-config file

Project 2:
`ts-node -T ./playground/monorepo_alt/samples/project2/fuse`
This have baseurl in extended tsconfig file

atm its bugging out on 1 of the packages, dunno why atm
2 last errors Ive added on purpose to see it works like intended
image

### CODE REVIEW

**Comment by vegarringdal** [comment]
@nchanged 
Think I have it now.


Summary:
* will combine better with extended
* moved the static compiler values you add after we combine the extened so we know nothing overrides them
* added more test for for baseURL/basepath/combine
* added vscode launch for debugging

**Comment by vegarringdal** [comment]
@nchanged 
Have you had time to look at this ?

**Comment by nchanged** [comment]
@vegarringdal please resolve the conflicts, I can't merge it

**Comment by vegarringdal** [comment]
I will try. need to pull changes.
I suck at git...

**Comment by vegarringdal** [comment]
@nchanged 
Looks like I fixed it :-)

**Comment by nchanged** [comment]
@vegarringdal https://www.gitkraken.com/ that will help you =) You need to rebase

**Comment by vegarringdal** [comment]
just added remote and did a git pull, thne I just comittet the merge conflict files

**Comment by vegarringdal** [comment]
but yes, think I will install that tool

**Comment by nchanged** [comment]
@vegarringdal you can squash some commits too. So it looks better

**Comment by vegarringdal** [comment]
I though you would just squash all maybe to 1 commit :-)
I can try, Ill have a look it.
Git isnt my strung side, but I should start to learn more.
maybe I should do more PR so I get practise 😂

**Comment by nchanged** [comment]
@vegarringdal I can rebase, then the order and all of your commits will be kept

File: playground/server/fuse.ts [hunk 1]
```diff
@@ -1,41 +0,0 @@
-import { fusebox, sparky } from '../../src';
-
-class Context {
```
Comment (nchanged) [inline playground/server/fuse.ts]: why was this file removed?

File: src/analysis/fastAnalysis.ts [hunk 2]
```diff
@@ -53,16 +53,16 @@ export interface IStatementReplaceAble {
 }
 /**
  * AN extremely simple and reliable module extractor based on one very well
- * optmised Regular expression
+ * optmimised Regular expression
```
Comment (nchanged) [inline src/analysis/fastAnalysis.ts]: why are we seeing this commits? =)

File: src/config/IPublicConfig.ts [hunk 3]
```diff
@@ -20,7 +20,7 @@ export interface IPublicConfig {
   useSingleBundle?: boolean;
   dependencies?: {
     include?: Array<string>;
-    ignorePackages?: Array<string>;
+    ignore?: Array<string>;
```
Comment (nchanged) [inline src/config/IPublicConfig.ts]: What's happening here? It should be `ignorePackages`

**Comment by vegarringdal** [comment]
hmm, looks like I messed up the merge conflict somehow

**Comment by vegarringdal** [comment]
I  manged to really mess up duing that merge, just close this and Ill make a new PR.
will install gitkraken, so I get a better tool to see git commits when doing operations like updating local fork etc
